cli/cmd/top.go

```diff
@@ -328,25 +335,30 @@ func stripPort(address string) string {
 	return strings.Split(address, ":")[0]
 }
 
-func renderHeaders() {
+func renderHeaders(withSource bool) {
 	tbprint(0, 0, "(press q to quit)")
 	x := 0
 	for i, header := range columnNames {
+		if i == 0 && !withSource {
```

Comment:
consider:
```go
if header == "Source" && !withSource {
```

Reply:
I think this is kinda brittle either way (brittle to changing the header string vs brittle to reordering the columns) so I'd lean towards keeping it the way it is.